source/administration/config-settings.rst

```diff
@@ -2702,6 +2702,14 @@ Obtain this value by registering Mattermost as an application in your Microsoft
 | This feature's ``config.json`` setting is ``"Secret": ""`` with string input.                                                                                        |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Directory (tenant) ID
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Obtain this value by registering Mattermost as an application in your Microsoft or Office account. 
```

Comment:
Technically, a Directory (tenant) ID is the ID of the AAD directory in which you created the application where a tenant represents an organization in Azure AD (so if MM is the org (tenant in AAD) it can have more than one application registered in it; "in your Microsoft or Office account" could be a bit vague since this is a cloud-specific term.

Reply:
Thanks for the feedback. How about: "This value is the ID of the application's AAD directory."?